config/locales/es.yml

```diff
@@ -48,8 +114,42 @@ es:
   pagination:
     next: Próximo
     prev: Anterior
+  remote_follow:
+      acct: Entra tu usuario@dominio que tu quieres seguir desde
+      missing_resource: No se ha encontrado la URL requerida para tu cuenta
+      proceed: Proceder a seguir
+      prompt: 'Tu estas apunto de seguir a:'
   settings:
+    authorized_apps: Aplicaciones Autorizadas
+    back: Volver a Mastodon
     edit_profile: Editar perfil
+    export: Exportar información
     preferences: Preferencias
+    settings: Ajustes
+    two_factor_auth: Two-factor Authentication
+  statuses:
+    open_in_web: Abrir en la web
+    over_character_limit: limite de carácteres, %{max} excedidos
```

Comment:
max is the number of maximal characters, not the quantity in excess, are you sure of the comma so?